modules/op-resolver-hub-config.adoc

```diff
@@ -33,9 +33,20 @@ spec:
       default-artifact-hub-pipeline-catalog: tekton-catalog-pipelines # <3>
       defailt-kind: pipeline # <4>
       default-type: tekton # <5>
+      tekton-hub-api: "https://my-custom-tekton-hub.example.com" # <6>
+      artifact-hub-api: "https://my-custom-artifact-hub.example.com" # <7>
 ----
 <1> The default {tekton-hub} catalog for pulling a resource.
 <2> The default {artifact-hub} catalog for pulling a task resource.
 <3> The default {artifact-hub} catalog for pulling a pipeline resource.
 <4> The default object kind for references.
 <5> The default hub for pulling a resource, either `artifact` for {artifact-hub} or `tekton` for {tekton-hub}.
+<6> The {tekton-hub} API used, if the `default-type` option is set to `tekton`.
+<7> The {artifact-hub} API used, if the `default-type` option is set to `artifact`.
++
+[IMPORTANT]
+====
+If you set the `default-type` option to `tekton`, you must configure your own instance of the {tekton-hub} by setting the `tekton-hub-api` value.
+
+If you set the `default-type` option to `artifact` then the resolver uses the public hub API at https://artifacthub.io/ by default. You can configure your own {artifact-hub} API by setting the `artifact-hub-api` value.
+====
```

Comment:
@vdemeester I remember we do not support `artifact` in RH Openshift Pipelines. I think in that case we should not include  `artifact` configuration in our documentation.
WDYT?

Reply:
We do not ship a supported artifacthub product yes, but we can still document that they can do, don't we ?

Reply:
@mramendi WDYT?

Reply:
@vdemeester I think, we have to mention exclusively that, we support only the configuration part of `artifact` and we do not support `artifact` product.

Reply:
I agree with @jkandasa - let's document the setting, but for Artifact Hub, put in a note that Artifact Hub custom instances are not currently supported by Red Hat. And we should put a link to "Using Tekton Hub with Pipelines" in the Additional Resources, that's where we document setting up a custom Tekton Hub instance.